build/search-index.js

```diff
@@ -1,6 +1,6 @@
 const { popularities } = require("content");
 
-const getPopularity = (item) => popularities[item.url];
+const getPopularity = (item) => popularities[item.url] || 0;
```

Comment:
@Gregoor this little change is important. Without it, the sorting gets all wrong. There are lots of URLs that aren't in `popularities` so the `undefined` would mess with the sort. 
Yes I know this isn't really strictly related to "fixable flaws" but wha'evs.

Also, something to consider is that if you're searching for something that matches 3 docs that don't have a popularity score, we should consider a secondary sort key. Alphabetically maybe? Newness maybe?

Reply:
Good point!

I also realized that I also put the popularity into the metadata and initialized it there. Probably best to just read it from there.